react-frontend/src/components/CaseStudies/horizontalinfocard.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import 'antd/dist/antd.css';
+import { Card } from 'antd';
+import '../../css/card.css';
+
+function HorizontalInfoCard(props) {
+  var isRouteCard = false;
+  if (props.routePath != null) {
+    isRouteCard = true;
+  }
+
+  return (
+    <Card
+      className="cardBodyHorizontal"
+      hoverable
+      bordered={false}
+      cover={
+        <img className="cardImageHorizontal" alt="example" src={props.img} />
+      }
+    >
+      <div className="cardTextHorizontal">
+        <div className="cardTitleHorizontal">{props.title}</div>
+        <div className="cardDescriptionHorizontal">{props.description}</div>
+        <div className="cardLinkHorizontal">
+          {isRouteCard ? (
```

Comment:
Instead of using 'isroutecard ? ' at all we can use props.routePath ?  js treats null as falsy, but treats a string as truthy

Reply:
we may want to change this on the other cards that use the check